### PR TITLE
Adds bar login

### PR DESCRIPTION
**Pull request by JakeKeels**

### CODE REVIEW

File: routes/loginUser.js [hunk 1]
```diff
@@ -17,9 +17,9 @@ const loginUser = async (request, db_connection) => {
   return await new Promise((resolve, reject) => {
 
     const sql_query = `
-            select users_id from users_pass where users_id = (
-                select id from users where username = '${username}'
-            ) and password = '${password}' group by users_id
+            select user_id from user_pass where user_id = (
+                select user_id from users where username = '${username}'
+            ) and password = '${password}' group by user_id
```
Comment (anthonymesa) [inline routes/loginUser.js:22]: This update does not match the current names for the tables and table values in the database

File: routes/loginBar.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ */
+
+const { Request } = require("tedious");
+const { createResponse } = require('../response')
+
+const loginUser = async (request, db_connection) => {
+
+  const username = request.params.username;
+  const password = request.params.password;
+  let user_id = null;
```
Comment (anthonymesa) [inline routes/loginBar.js:15]: maybe this should be bar_id?

**Review by anthonymesa — CHANGES_REQUESTED** [review]
Great job! only some minor changes!

**Comment by JakeKeels** [comment]
Fixes issues. Fixes issue #39.

**Pull request merged**